lib/ansible/cli/galaxy.py

```diff
@@ -170,6 +172,30 @@ def validate_signature_count(value):
     return value
 
 
+@dataclass
+class RoleDistributionServer:
+    _api: t.Union[str, None]
```

Comment:
api_servers will always be a list of at least one GalaxyAPI and _api is None until `api` is called.
```suggestion
    _api: t.Union[GalaxyAPI, None]
```